workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePage.tsx

```diff
@@ -236,21 +236,22 @@ export const WorkflowInstancePage = ({
             />
             <Grid container item justifyContent="flex-end" spacing={1}>
               <Grid item>
-                <Tooltip
-                  title="user not authorized to abort workflow"
-                  disableHoverListener={permittedToUse.allowed}
-                >
-                  <Button
-                    variant="contained"
-                    disabled={!permittedToUse.allowed || !canAbort}
-                    onClick={toggleAbortConfirmationDialog}
-                    className={classes.abortButton}
+                {permittedToUse.allowed && canAbort && (
```

Comment:
please make the button available and disabled with tooltip when no permission
it should not appear only depending on status